keyboards/xdboards/recon/keyboard.json

```diff
@@ -0,0 +1,229 @@
+{
+    "manufacturer": "XD Boards",
+    "keyboard_name": "xdboards/recon",
+    "maintainer": "Xanimos",
+    "development_board": "elite_pi",
+    "pin_compatible": "elite_c",
```

Comment:
This combination logically doesnt make sense.

Either,
1. Its configured as a elite-c using AVR pins, and the use of development board is incorrect

or
2. Its configured as a elite-pi, and the use of pin_compatible is incorrect